package.json

```diff
@@ -27,9 +27,9 @@
     "react-dom": "^0.14.7"
   },
   "devDependencies": {
+    "babel-core": "^6.4.5",
     "babel-loader": "^6.2.2",
     "babel-preset-es2015": "^6.3.13",
-    "babel-preset-es2015-loose": "^7.0.0",
```

Comment:
I thought I saw that without loose mode babel does some slow things in the name of spec-correctness. Babel has such crappy documentation that I can't find an authoritative source for the tradeoffs.